### DISCUSSION TITLE
Gopro 11 Construction Timelapse Best Approach?

### DISCUSSION THREAD

**Discussion by pabsTDM** [gopro/labs#393] [Q&A]
So I am looking to use a gopro 11 connected to a  50amp car battery of sorts which will be powered via a solar panel.

Construction project will be over 11 months total. I am hoping to capture 1 image ever hour from 6am to 6pm. I was planning to wire in a 4G connect with wifi to allow the camera to upload so I can remotely check its status and plan to download images from camera SD once a month.

This is my current code which seams to be working great until I plug it into USB power and then it continuously power cycles?
18:00!06:00NmNPp.2!S!10E!3600R

My questions are?
Am i better off having the camera power cycle ever hour between the image or just keep the camera on for the 12hrs period?
Why does it contentiously power cycle when i connect usb power?
also I am open to suggestions

**Comment by pabsTDM** [comment]
Also have tried this but also power cycles when on usb power? 

!06:00NfWp.3600!S!43200E!U!1R

**Comment by dnewman-gpsw** [comment]
For reliability, just use the default photo time lapse mode set to a one hour interval.  Do not upload, do not shutdown the camera, as for eleven months reliability overrides fancy configurations.

**Reply by pabsTDM** [reply]
This would not be ideal as I do not require images outside the 6am to 6pm period. Secondly I would like to use the upload feature as a remote monitoring too to confirm if it is work.

**Reply by dnewman-gpsw** [reply]
As you want this to run for 11 months, you should use the stock mode and/or an external timer.  Other is it only reliable for a few days.

**Comment by dnewman-gpsw** [comment]
The second solution is it use a external timer, with boot commands.  However I have yet to find the ideal external timer.

**Comment by standpointmedia** [comment]
Hi @dnewman-gpsw - what about something like the Camdo BlinkX or UpBlink as an external timer?

**Reply by pabsTDM** [reply]
Hi Standpointmedia, though UpBlink would be a solution while price it for this project it has worked out cheaper and in my opinion better features to use a tikee 3 camera.

Have decided to forego the gopro foe this project and purchased a tikee 3 pro system to complete the project which will be much more reliable by the sound of things.

**Reply by standpointmedia** [reply]
Yeah, I have a few Tikees, great hardware, but the software is getting very restrictive and expensive though.

**Reply by pabsTDM** [reply]
We have just purchase the pro version. Any advise for a 11 month long timelapse? Using this hardware?

We will be using 4g connection, monthly site visit.  And the story telling $65 /35 a month subscription.

**Reply by standpointmedia** [reply]
You should be fine with that set up, I have 3 running right now, one for over a year. The online software is a bit confusing and takes some time to get used to. You'll probably be better off on in the end making a timelpase from the JPGs on the camera.
